scripts/preprocess.py

```diff
@@ -0,0 +1,29 @@
+import os
+from argparse import ArgumentParser
+from shutil import copytree, rmtree
+
+
+def fix_naming(dataset_path: str):
+    train = dataset_path + "java-med-psi.train.c2s"
+    val = dataset_path + "java-med-psi.val.c2s"
+    os.remove(train)
+    os.remove(val)
+
+
+def preprocess(project_path: str):
+    project_name = os.path.basename(os.path.normpath(project_path))
+    dataset_path = "../datasets/" + project_name + "/java-med-psi/"
```

Comment:
Moreover, let's remove some constants to the `utils` file. For example, the path to the datasets directory.